pkg/logging/logger.go

```diff
@@ -271,7 +271,7 @@ func (l logrusEntryWrapper) Logf(level logrus.Level, format string, args ...inte
 }
 
 func (*logrusEntryWrapper) IsTracing() bool {
-	return logrus.IsLevelEnabled(logrus.TraceLevel)
+	return defaultLogger.IsLevelEnabled(logrus.TraceLevel)
```

Comment:
Neat!  I remember this line having a long and worrying history :-/

Reply:
now I'm worried from the amount of logging while we trace.